includes/class-send-list.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/**
+ * Newspack Newsletters Send List
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Newsletters;
+
+use Newspack_Newsletters;
+use WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class used to represent one Send_List.
```

Comment:
Maybe too verbose.. but just trying to add some rationale for our future selves... 

```suggestion
 * Class used to represent one Send_List.
 * 
 * A Send List is any list that can be used as a segment of contacts that you can send a Newsletter campaign to
 * The difference between Send_List and Subscription_List is that Send Lists are all the existing lists, audiences, tags, groups, segments, etc in the provider. We don't store them locally. Some of these can also be Subscription Lists, but not all are.
```